redis/__init__.py

```diff
@@ -1,5 +1,6 @@
 import sys
 
+from redis.backoff import get_default_backoff
```

Comment:
Can we change get_default_backoff to default_backoff? It's return implies a get

Reply:
changed